### DISCUSSION TITLE
Quando Client  recebe o JSON é suprimido o undeline "_"

### DISCUSSION THREAD

**Discussion by ademarrs** [HashLoad/horse#303] [Q&A] [answered]
Nossos campos no banco de dados costuma ter um undeline "_" no nome, por exemplo _**pro_descricao**_, quando é feito um GET para recuperar os dados de um registros o JSON vem com os nomes dos campos sem o undeline.
Tem alguma configuração que podemos ajustar pra não suprimir o undeline?

**Comment by viniciussanchez** [comment, marked as answer]
Boa tarde, isso não tem nada a ver com o Horse.
Isso vai ser algo da biblioteca de serialização de JSON que você está utilizando. Por exemplo, o DataSet.Serialize. Caso seja ele, sim, tem como configurar isso no DataSet-Seralize, e não no Horse.

**Reply by ademarrs** [reply]
Certo, estamos usando o DataSet-Seralize, como podemos ajustar a configuração?

**Reply by viniciussanchez** [reply]
Está lá na documentação....
![image](https://user-images.githubusercontent.com/16382981/186983788-302d02f4-1735-497b-b7b4-b938c355744c.png)

**Reply by ademarrs** [reply]
Deu certo, 
Obrigado.
